src/network/clientpackethandler.cpp

```diff
@@ -1209,8 +1209,14 @@ void Client::handleCommand_HudSetParam(NetworkPacket* pkt)
 
 	if (param == HUD_PARAM_HOTBAR_ITEMCOUNT && value.size() == 4) {
 		s32 hotbar_itemcount = readS32((u8*) value.c_str());
-		if (hotbar_itemcount > 0 && hotbar_itemcount <= HUD_HOTBAR_ITEMCOUNT_MAX)
+		if (hotbar_itemcount > 0 && hotbar_itemcount <= HUD_HOTBAR_ITEMCOUNT_MAX) {
+			u32 main_size = player->inventory.getList("main")->getSize();
```

Comment:
`getList("main")` needs checking against `nullptr` to avoid segfaults when there's no "main" inventory or the command arrived before the client is fully initialized.